iml-warp-drive/src/error.rs

```diff
@@ -12,7 +12,7 @@ pub enum ImlWarpDriveError {
     ImlRabbitError(ImlRabbitError),
     ImlManagerClientError(ImlManagerClientError),
     TokioPostgresError(iml_postgres::Error),
-    DbError(DbError),
+    DbError(Box<DbError>),
```

Comment:
Why is this?

Reply:
To mitigate `clippy::large_enum_variant`. One way is to wrap large variant into `Box` or just supress the warning.

Reply:
The enum will always be sized to it's largest variant. By putting the value in a box, the item is moved to the heap so it's just a pointer now.

Reply:
Ok, I got it :)